pkg/restclient/volume.go

```diff
@@ -183,3 +183,11 @@ func (c *Client) VolumeReset(volname string, req api.VolOptionResetReq) error {
 	url := fmt.Sprintf("/v1/volumes/%s/options", volname)
 	return c.del(url, req, http.StatusOK, nil)
 }
+
+//VolumeProfileInfo retireves the stats about different file operations performed on a volume
+func (c *Client) VolumeProfileInfo(volname string, option string) ([]api.BrickProfileInfo, error) {
+	var volumeProfileInfo []api.BrickProfileInfo
+	url := fmt.Sprintf("/v1/volumes/%s/profile/%s", volname, option)
+	err := c.get(url, nil, http.StatusCreated, &volumeProfileInfo)
```

Comment:
question:
do we need to return HTTP status code 201 or 200  for ```HTTP GET```  request?

Reply:
We need to return 201 for this request, GD2 server returns 201 for this request.

Reply:
change in server code, 201 is "Status Created", which is used when a API creates a new resource. Change it 200(`http.StatusOK`)

Reply:
as @aravindavk said if we are creating a resource we need to use ```POST ``` and return 201. ```GET``` is meant to query some info from the server side it should return 200

Reply:
Changed to StatusOK.